jenkins-pr-collector.go

```
@@ -434,65 +558,325 @@ func getCommitStatus(commits struct {
 	return commits.Nodes[0].Commit.StatusCheckRollup.State
 }
 
+// Add function to save partial data
+func savePartialData(prs []PullRequest, cursor string, outputFile string) error {
```

golangci-lint (1.62.2)

func `savePartialData` is unused

(unused)